package.json

```diff
@@ -72,6 +72,7 @@
     "@vscode/sqlite3": "5.1.4-vscode",
     "@vscode/sudo-prompt": "9.3.1",
     "@vscode/vscode-languagedetection": "1.0.21",
+    "@vscode/windows-mutex": "^0.4.4",
```

Comment:
Is it intentional that we are now making it "non-optional"?

Reply:
Yep. It will be available on all platforms so that the types can be read properly during compile time. We have run-time checks to make sure we don't call into the code during run-time.

Reply:
Making it non-optional would increase the bundle size for non-windows distributions so we might want to filter them out.

Reply:
Closing this PR until we have a better solution to the implementation of `cleanNodeModules`

Reply:
Resolving this conversation for I have implemented OS-specific modules.